.drone.yml

```diff
@@ -16,8 +16,13 @@ pipeline:
       - go test -v -cover $(go list ./... | grep -v /vendor)
       - go build $(go list ./... | grep -v /vendor)
 
-matrix:
-  GO_VERSION:
-    - latest
-    - "1.8"
-    - "1.7"
+  test-e2e:
+    image: golang:1.8
+    environment:
+      - KETO_ASSETS_DIR=/ketoassets
+      - CLUSTER_NAME=drone-build-${DRONE_BUILD_NUMBER}
+    commands:
```

Comment:
Maybe add a requirement and a check for `E2E=true` parameter? Right now e2e test gets kicked off when `event=deployment`.